### ISSUE TITLE
Zooming only images, not links

### ISSUE THREAD

**Issue by mwld** [unarj/hooverZoom#3]
Hi there!

Thanks for the great plugin! I'm using it now because Thumbnail Zoom Plus (https://addons.mozilla.org/de/firefox/addon/thumbnail-zoom-plus/) stopped working with the update of Firefox to version 57.

One question I have because here is where hooverZoom is different from Thumbnail Zoom Plus: Is it possible to restrict the zooming to images and exclude links? Meaning I want a zoomed image only to show up when having my mouse on an image (which does not necessarily need to be part of a link). I don't need a preview image when hovering a link so to say.

Is that currently possible?

**Comment by unarj** [comment]
that would take a complete rewrite since links are the hooks that this add-on uses to do it's thing.  in my experience images that aren't linked are already displayed full size, links to other images are where the real opportunity to do some work and save the click to show you media.

anyways, sorry but it looks like you're looking for a different add-on.

**Comment by mwld** [comment]
Yeah, you are right. I was actually looking for a replacement of Thumbnail Zoom Plus (https://addons.mozilla.org/de/firefox/addon/thumbnail-zoom-plus/) which has not yet been ported to a WebExtension. But I now found an addon that got ported from Chrome and does what I am looking for: Hover Zoom+ (https://addons.mozilla.org/en-US/firefox/addon/hoverzoom-plus/).
